.github/workflows/release-editor-kit.yml

```
@@ -0,0 +1,61 @@
+          cache: pnpm
+
+      - name: Make build script executable
+        run: chmod +x ./scripts/build-editor-preload.js
+
+      - name: Build editor-preload package
+        run: |
+          USE_NPM_ARG=""
+          if [ "${{ github.event.inputs.use_npm }}" == "true" ]; then
+            USE_NPM_ARG="--use-npm"
+          fi
+          
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)